repondeur/zam_repondeur/fetch/an/liasse_xml.py

```diff
@@ -47,18 +49,22 @@ def import_liasse_xml(xml_file: IO[bytes]) -> List[Amendement]:
         raise ValueError(message)
 
     uid_map: Dict[str, Amendement] = {}
+    errors = []
     for child in root:
-
         if extract_from_node(child, "etat") == "A déposer":
             num_long = extract_from_node(child, "numeroLong")
             logger.warning(f"Ignoring amendement {num_long} (à déposer)")
             continue
 
         uid = child.find(f"./{NS}uid").text
-        amendement = _make_amendement(child, uid_map)
-        uid_map[uid] = amendement
+        try:
+            amendement = _make_amendement(child, uid_map)
+            uid_map[uid] = amendement
+        except Exception as exc:
+            logger.exception(f"Failed to import amendement {uid} from liasse")
+            errors.append((uid, str(exc)))
```

Comment:
Do we still need the exception trace in errors?

Reply:
I think it can help us understand what went wrong exactly.

What do you think?

Reply:
Alright, I don't get where it is actually sent from here?

Reply:
See b7f3107